server/server/src/sync/mod.rs

```diff
@@ -50,3 +50,9 @@ impl SyncImportError {
         }
     }
 }
+
+fn is_sync_disabled(connection: &StorageConnection) -> Result<bool, RepositoryError> {
```

Comment:
Move close to sync_disabled so that its easier to see that the key value store IO matches... (see comment below)

Reply:
Good idea, although I had to change a bit to make service provider available (also made sure we use service provider for SettingsService): a91bf560d772c7afc349f8f3e76019103920e7a3